### PR TITLE
add new register page

### PR DESCRIPTION
**Pull request by mnatour2**
image

### CODE REVIEW

File: Register/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="styleSheet.css" />
+    <title>Register</title>
+</head>
```
Comment (MajdTaweel) [inline Register/index.html]: Use the same font used in the login page.

File: Register/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  justify-content: center;
+  font-size: small;
+}
+
+#login {
+  font-size: small;
+  display: flex;
+  justify-content: center;
+  margin: 18px;
+}
```
Comment (MajdTaweel) [inline Register/stylesheet.css]: Duplicated. Use the same stylesheet used for the login page.

**Review by MajdTaweel — CHANGES_REQUESTED** [review]
The checkbox and the text after it are not centered.

File: Register/stylesheet.css (hunk 2 above)
Comment (mnatour2) [reply Register/stylesheet.css]: fixed

**Pull request merged**